Tests/TestingApps/PaywallsTester/PaywallsTester/Configuration.swift

```diff
@@ -6,34 +6,64 @@
 //
 
 import Foundation
+import RevenueCat
 
-enum Configuration {
+class Configuration: ObservableObject {
+    static let shared = Configuration()
 
-    #warning("Configure API key if you want to test paywalls from your dashboard")
+    // This is modified by CI:
+    private static let apiKeyFromCIForTesting = ""
+    private static let apiKeyFromCIForDemos = ""
 
-    // Note: you can leave this empty to use the production server, or point to your own instance.
-    static let proxyURL = ""
-    static let apiKey = ""
+    @Published private(set) var currentMode: Mode
 
-    static let entitlement = "pro"
+    let entitlement = "pro"
 
-}
+    #warning("Configure API key if you want to test paywalls from your dashboard")
+    // Note: you can leave this empty to use the production server, or point to your own instance.
+    private let proxyURL = ""
+    private let apiKey = ""
 
-extension Configuration {
+    enum Mode {
+        case custom, testing, demos
+    }
 
-    static var effectiveApiKey: String = {
-        return Self.apiKey.nonEmpty ?? Self.apiKeyFromCI
-    }()
+    private init() {
+        currentMode = apiKey.isEmpty ? .testing : .custom
```

Comment:
Explicit `self` per our style guide:
```suggestion
        self.currentMode = apiKey.isEmpty ? .testing : .custom
```